deno_dist/middleware/body-limit/index.ts

```diff
@@ -25,42 +33,57 @@ type BodyLimitOptions = {
  * )
  * ```
  */
-export const bodyLimit = (options: BodyLimitOptions): MiddlewareHandler =>
-  async function bodyLimit(c, next) {
-    if (!c.req.raw.body) return next()
+export const bodyLimit = (options: BodyLimitOptions): MiddlewareHandler => {
+  const onError: OnError = options.onError || ((c) => c.text('413 Request Entity Too Large', 413))
```

Comment:
It is better to make `'413 Request Entity Too Large'` as a constant to reuse in line 66.

Reply:
@yusukebe Thank you!

refactored in 2a326ca

The message has been changed to "Payload Too Large" in accordance with the RFC.
https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.11

Reply:
Great 👍